src/machine/behavior/is/float/fcmp.rs

```diff
@@ -1,6 +1,17 @@
 use machine::state::State;
 
-pub fn fcmp(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn fcmp(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let op1: f64 = state.gpr[y].into();
+    let op2: f64 = state.gpr[z].into();
+
+    // Execute and store result
+    if op1 - op2 > 0.0 {
+        state.gpr[x] = 1i64.into();
+    } else if op1 - op2 < 0.0 {
+        state.gpr[x] = (-1 as i64).into();
+    } else {
+        state.gpr[x] = 0i64.into();
+    }
```

Comment:
That's funny... I was just writing about that. The short answer is: Not really. The long answer will follow soon...